pkg/requests/requests_test.go

```diff
@@ -86,23 +87,25 @@ func TestRequestUnparsedResponseUsingAccessTokenParameter(t *testing.T) {
 		}))
 	defer backend.Close()
 
-	response, err := RequestUnparsedResponse(
+	ctx := context.TODO()
```

Comment:
Use Background in tests as TODO suggests we are going to replace it at some point in the future
```suggestion
	ctx := context.Background()
```